src/components/Subnavigation/Subnavigation.mdx

```diff
@@ -0,0 +1,34 @@
+import { Preview, Story, ArgsTable, PRIMARY_STORY } from '@storybook/addon-docs/blocks';
+import Subnavigation from './Subnavigation.vue';
+
+# Subnavigation
+
+The subnavigation component is used for displaying a secondary, horizontal navigation.
+
+<Preview>
+  <Story id="components-subnavigation--primary" />
+</Preview>
+
+## How to Use
+
+### Subnavigation
+
+The subnavigation renders a semantic `nav` element with a `ul`. It is made to work with the `subnavigation-item` component. However, you can also write custom components for its slot as long as the root element is an `li` (for accessibility).
+
+Example of using this component in a template:
+
+```html
+<subnavigation v-bind="$props">
```

Comment:
Nitpick, can we remove the `v-bind` here since that's only for the story itself?